elide-contrib/elide-dynamic-config-helpers/pom.xml

```diff
@@ -115,6 +112,18 @@
             <artifactId>handlebars</artifactId>
             <version>${handlebars.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.mdkt.compiler</groupId>
+            <artifactId>InMemoryJavaCompiler</artifactId>
+            <version>${mdkt.compiler.version}</version>
+            <optional>true</optional>
```

Comment:
Optional=true is really meant for autoconfigure packages - not a core elide package.